scripts/index.js

```diff
@@ -21,6 +21,19 @@ const datapackageJson = async (url, options={}) => {
   return body
 }
 
+// const pageLoadTime = async (url) => {
```

Comment:
@Mikanebu I did not request to write this back and comment. but just asked the question why this was removed. That not necessarily mean you should write back, but just answer the question. Even now when this is commented, why it is commented? why was it here? I don't see any commit with message explaining this. That's why I asked and not because I wanted it back

Reply:
It is my fault, I forgot to explain this in the first place. I uncommented because it was not used in my test cases, so I decided that it is doing extra unnecessary work.

Reply:
cool, now I'm clear